libs/user-identity-service/src/test/java/uk/gov/di/ipv/core/library/service/UserIdentityServiceTest.java

```diff
@@ -274,9 +261,7 @@ void checkNameCorrelationWithMissingNameWithAddressCredentialsForReturnTrue() th
         List<VcStoreItem> vcStoreItems =
```

Comment:
I don't understand the name of this test so not sure what it is testing, but this is the change that was made on the main branch (removing the passport VC).

Reply:
I think the intent is that name correlation is happy to skip address credentials (which don't include a name)